src/models/BaseModel.py

```diff
@@ -80,39 +78,45 @@ def preprocess_data(self):
         print('Pre-processing begins')
         # Description: consisting of 5 features existence of words below in the description:
         # 'excellent', 'graduate', 'immediate', 'junior', 'urgent'
-        onehot_encoded_desc_words = get_one_hot_encoded_words(self.description_feature)
+        (onehot_encoded_desc_words, col_names_1) = \
```

Comment:
thought about it too, wouldn't mind changing, this was more straightforward because of

        concatenated_column_names = np.concatenate((
            col_names_1,
            col_names_2,
            col_names_3,
            col_names_4,
            col_names_5,
            col_names_6,
            col_names_7,
...